projects/demo/src/app/datagrid/sorting/sorting.ts

```diff
@@ -23,10 +29,19 @@ export class DatagridSortingDemo {
   sortOrder: ClrDatagridSortOrder = ClrDatagridSortOrder.UNSORTED;
 
   pokemonComparator = new PokemonComparator();
+  userCreationComparator = new UserCreationComparator();
 
   constructor(inventory: Inventory) {
     inventory.size = 10;
     inventory.reset();
     this.users = inventory.all;
   }
+
+  setSortOrder(newOrder: ClrDatagridSortOrder) {
+    this.sortOrder = newOrder;
+  }
+
+  clrDgSortOrderChange($event: ClrDatagridSortOrder) {
+    console.log($event);
```

Comment:
I think this is leftover debugging code. It's only applies to one column, and the log message is not particularly helpful. Please remove.

Reply:
corrected in commit 3684b86